backend/test/scans.test.ts

```diff
@@ -368,3 +376,25 @@ describe('scheduler invoke', () => {
     expect(scheduler).toHaveBeenCalledTimes(0);
   });
 });
+
+describe('run scan', () => {
+  it('run scan should change lastRun to null', async () => {
```

Comment:
Can you add a test to ensure that only globalAdmins can access this route? (so the test can just call the endpoint with a "globalView" user and ensure it returns a 403)

Reply:
Are global admins the only ones allowed to run a scan on demand?

Reply:
Yeah, at the moment, we should do that.